cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -35,9 +35,9 @@ template <typename settings>
 Prover<settings>::Prover(std::shared_ptr<waffle::proving_key> input_key, const transcript::Manifest& input_manifest)
     : n(input_key == nullptr ? 0 : input_key->n)
     , transcript(input_manifest, settings::hash_type, settings::num_challenge_bytes)
-    , proving_key(input_key)
+    , key(input_key)
```

Comment:
`proving_key` is better, but `key` is used in a test, and my policy is to change Honk stuff.